src/components/card/card.stories.mdx

```diff
@@ -65,20 +99,13 @@ const singleDemoStory = (args) => {
 
 A card contains content and optionally an action summarizing a single piece of content, for example an article, talk or case study.
 
+If your card represents an `article` and includes a heading, it's a good idea to provide a `heading_id` as well. This allows us to use `aria-labelledby` to improve the experience of navigating to individual cards via the [VoiceOver rotor](https://support.apple.com/en-us/HT204783) (and potentially other assistive interfaces as well).
```

Comment:
Thanks for including a link, @tylersticka! 

I hadn't actually tried accessing the "articles" rotor menu on an iOS device and I was surprised in two ways:

1. The "articles" menu is not on by default on iOS VoiceOver (it _is_ on by default on macOS VoiceOver)
2. When I navigated by articles via iOS VoiceOver on our live [site](https://cloudfour.com/thinks/), it _does_ read out the heading already. 🤯 This experience is _different_ from macOS VoiceOver. macOS VoiceOver _does not_ read the heading and instead only lists "article" as the [original issue](https://github.com/cloudfour/cloudfour.com-patterns/issues/1539) described.

This made me wonder if the linked article didn't quite make sense since the experience was already good on iOS VoiceOver.

I thought, "I'll find a macOS VoiceOver 'articles menu' rotor link to suggest instead!" but I can't find one. 🤔 

That's a lot of words, I ended with these questions:
1. This is macOS VoiceOver rotor-specific, so maybe we mention that?
2. Can we find a macOS VO-specific article, otherwise, don't link to anything?

Let me know if I didn't make sense. 🙃

Reply:
@gerardo-rodriguez Is this better? 26c1e7c

Reply:
That works great, @tylersticka, thanks! 😄